packages/next/README.md

```diff
@@ -15,13 +15,46 @@
   </a>
 </p>
 
+## The React Framework
+
+Next.js is used by the world's leading companies and independent developers. 
+
+* **Pre-Rendering** - Statically generated and server-rendered React applications have never been easier.
+* **Static Exporting** - No need to learn a new framework. Exporting a static site with Next.js is as easy as a single command.
+* **CSS-in-JS** - Next.js comes with `styled-jsx` included, but it also works with every CSS-in-JS solution you know and love.
+* **Built-in TypeScript support** - Next.js provides an integrated TypeScript experience out of the box, similar to an IDE.
+* **Zero Configuration** - Automatic code splitting, filesystem based routing, hot code reloading and universal rendering. All included.
+* **Fully Extensible** - Complete control over Babel and Webpack. Customizable server, routing and next-plugins.
+* **Ready for Production** - Optimized for a smaller build size, faster dev compilation and dozens of other improvements.
+
 ## Getting Started
 
-Visit <a aria-label="next.js learn" href="https://nextjs.org/learn">https://nextjs.org/learn</a> to get started with Next.js.
+Creating a new app with Next.js is easy with the `create-next-app` CLI. Create Next App allows you to create a Next.js app with a single command and set up everything for you, enabling you to get started within minutes.
+
+```sh
+npx create-next-app
+# or
+yarn create next-app
+```
+
+For more details on getting started with Next.js, including how to manually create a new app, <a aria-label="Next.js getting started instructions" href="https://nextjs.org/docs/getting-started#setup">follow the instructions here.</a> 
```

Comment:
Not required, better to link to https://nextjs.org/learn/basics/create-nextjs-app for more detailed instructions.

Reply:
@MehediH my feedback here is still unresolved

Reply:
Sorry I just committed the changes.